.github/workflows/demo_deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - remove-sensitive-info
```

Comment:
Let's not forget to remove this before merging

Reply:
done